### PR TITLE
Improve the documentation for checking out MOOSE 

### PR DESCRIPTION
**Pull request by bwspenc**
ref #2

### CODE REVIEW

File: doc/content/getting_started/RunningBlackBear.md [hunk 1]
```diff
@@ -72,17 +81,17 @@ cd ~/projects/blackbear/
...
 
-Then update the MOOSE submodule:
+Then update MOOSE:
 
 ```bash
-cd ~/projects/blackbear/
-git submodule update
+cd ~/projects/moose/
+git pull
 ```
```
Comment (sapitts) [inline doc/content/getting_started/RunningBlackBear.md]: Would it be prudent to include some information about git clean before rebuilding libmesh?

File: doc/content/getting_started/RunningBlackBear.md [hunk 2]
```diff
@@ -32,20 +32,29 @@ cd ~/projects/
...
+BlackBear requires a checkout of the MOOSE repository to link against. By default, the BlackBear
+build process looks for that in a directory called `moose` located in the same directory
+that contains the `blackbear` directory. Clone the MOOSE repository using the following command:
 
 ```bash
-cd ~/projects/blackbear/
-git submodule update --init
+cd ~/projects/
+git clone git@github.com:idaholab/moose.git
 ```
```
Comment (sapitts) [inline doc/content/getting_started/RunningBlackBear.md]: I'd recommend including an example of how the projects directory should look, with both moose and blackbear, or perhaps a line about verifying that blackbear and moose appear in the projects directory

File: doc/content/getting_started/RunningBlackBear.md [hunk 3]
```diff
@@ -72,17 +81,17 @@ cd ~/projects/blackbear/
...
 
-Then update the MOOSE submodule:
+Then update MOOSE:
 
 ```bash
-cd ~/projects/blackbear/
-git submodule update
+cd ~/projects/moose/
+git pull
```
Comment (sapitts) [inline doc/content/getting_started/RunningBlackBear.md]: Are we recommending git pull or git rebase upstream/{devel|master}? I had thought that the moose guys were recommending git rebase, but I could be out of the loop these days

File: doc/content/getting_started/RunningBlackBear.md (hunk 2 above)
Comment (bwspenc) [reply doc/content/getting_started/RunningBlackBear.md]: Done

File: doc/content/getting_started/RunningBlackBear.md (hunk 3 above)
Comment (bwspenc) [reply doc/content/getting_started/RunningBlackBear.md]: Done

File: doc/content/getting_started/RunningBlackBear.md (hunk 1 above)
Comment (bwspenc) [reply doc/content/getting_started/RunningBlackBear.md]: I'd rather keep this simple. That isn't strictly necessary -- I never do it. They could accidentally delete files they care about if they do a `git clean1.

**Pull request merged**